azure-pipelines.yml

```diff
@@ -53,15 +53,14 @@ jobs:
         - python-devel
 
     envs:
-      - macos: py36
-        name: py36_test
-        posargs: -n=1
+      - macos: py38
+        name: py38_test
```

Comment:
Why has this changed?

Reply:
Don't need it anymore.

Reply:
Oh you mean the Python version? We had too many 36 builds, I thought I would bump macos to a more modern version.